app/controllers/users_controller.rb

```diff
@@ -1,4 +1,25 @@
 class UsersController < ApplicationController
+  def show
+    @user = User.find_by id: params[:id]
```

Comment:
khi không tìm thây user nó sẽ báo lỗi : undefined method `name' for nil:NilClass